ros2/diagnostics/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>diagnostics</name>
+  <version>0.0.0</version>
+  <description>A diagnostics tutorial package</description>
```

Comment:
This package is not really a tutorial but more an example of how to publish diagnostics